src/compiler/file_uploader/uploader_identity.ts

```diff
@@ -0,0 +1,20 @@
+import {
+    addController,
+    generateIdentity,
+    getPrincipal,
+    identityExists
+} from '../../../dfx';
+
+export const AZLE_UPLOADER_IDENTITY_NAME = '_azle_file_uploader_identity';
```

Comment:
Should we make this an environment variable now just in case? And default to that name?

Reply:
sure